sprv/type/zif_aff_sprv_v1.intf.abap

```diff
@@ -0,0 +1,37 @@
+interface zif_aff_sprv_v1
+public .
+
+
+  types:
+    "! <p class="shorttext">Create a service provider</p>
+    "! Create a service provider
+    BEGIN OF ty_provdetails,
+      "! <p class="shorttext">Name</p>
+      "! Name of the Service Provider
+      "! $required
+      name TYPE c LENGTH 30,
+      "! <p class="shorttext">Original Language</p>
+      "! Original Language
+      "! $showAlways
+      org_lang TYPE c LENGTH 2,
```

Comment:
Why do you need the original language again? Is it different than the original language in the header?

Reply:
You are totally right this is redundant. I will remove the original language outside the header.